specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/examples/CreateOrUpdateWorkloadClassifierMax.json

```diff
@@ -0,0 +1,54 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "Default-SQL-SouthEastAsia",
+    "serverName": "testsvr",
+    "databaseName": "testdb",
+    "workloadGroupName": "wlm_workloadgroup",
+    "workloadClassifierName": "wlm_workloadclassifier",
+    "api-version": "2019-06-01-preview",
+    "parameters": {
+      "properties": {
+        "memberName": "dbo",
+        "label": "test_label",
+        "context": "test_context",
+        "startTime": "12:00",
+        "endTime": "14:00",
+        "importance": "high"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "memberName": "dbo",
+          "label": "test_label",
+          "context": "test_context",
+          "startTime": "12:00",
```

Comment:
Swagger & ARM team reviewers, what is the guidance for exposing "time of day" (i.e. just time, not date+time)?

Reply:
> Swagger & ARM team reviewers, what is the guidance for exposing "time of day" (i.e. just time, not date+time)?
@ryansbenson, what is the suggestion here?